pages/views/index.njk

```diff
@@ -1,5 +1,6 @@
 ---
 layout: landing
+description: Open-source UI component library that helps developers create modern-looking, effective, and user-friendly interfaces faster.
```

Comment:
pages/views/_includes/landing/marquee.njk: 7
"Open-source UI components library that helps developers create modern-looking, effective, and user-friendly interfaces faster."
The issue word "components" is written in plural form in on case and in single form in other case